packages/server/src/app/shared/admin/approval-queue.interface.ts

```diff
@@ -0,0 +1,56 @@
+import { PaginateResult } from 'mongoose';
+import { ApprovalQueue } from '@dragonfish/models/approval-queue';
+import { ContentKind, ContentModel } from '@dragonfish/models/content';
+import { JwtPayload } from '@dragonfish/models/auth';
+
+/**
+ * ## The Approval Queue Interface
+ *
+ * Implement this interface to be able to serve data to associated routes.
+ */
```

Comment:
Nitpick: Describe what this service does rather than what the interface is for. The people reading this docstring will be in a controller trying to figure out what an `IApprovalQueue` _does_. They don't really care if it's an interface or a class or whatever.